.travis.yml

```diff
@@ -1,35 +1,33 @@
-sudo: false
+sudo:
+    - false
 
-language: cpp
+language:
+  - cpp
 
 cache:
-  directories:
-    - odb
+    directories:
+        - odb
```

Comment:
Please use consistent indentation, the rest of the file uses 2 spaces per level.